client/views/wishCards.ejs

```diff
@@ -6,16 +6,15 @@
 	<!-- ** NOTE: CDN Bootstrap.js, Popper.js, and jQuery are moved to the bottom so the page loads faster-->
 	<%- include('./templates/header')-%>
 	<link rel="stylesheet" href="/public/css/search.css">
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/moment.min.js" integrity="sha512-qTXRIMyZIFb8iQcfjXWCO8+M5Tbc38Qi5WzdPOYZHIlZpzBHG3L3by84BBBOiRGiEb7KKtAOAs5qYdUiZiQNNQ==" crossorigin="anonymous"></script>
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.3.1/socket.io.js" integrity="sha512-AcZyhRP/tbAEsXCCGlziPun5iFvcSUpEz2jKkx0blkYKbxU81F+iq8FURwPn1sYFeksJ+sDDrI5XujsqSobWdQ==" crossorigin="anonymous"></script>
 	<script src="/public/clientJS/utils.js"></script>
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
 	<script>
-
-
 		<% for (var i = 0; i < wishcards.length; i++) { %>
-
+			<% if (moment(wishcards.isLockedUntil) > moment()) { %>
```

Comment:
moment comes here from the backend actually the script import is old code that I forgot to remove and there it's used anyway, it was more out of habit since moment has a better handling of timestrings. but yeah date works just fine here